src/dataframe/dataframe.jl

```diff
@@ -262,6 +262,9 @@ end
 
 Equivalent to `DataFrame(args...; copycols=false, kwargs...)`.
 
+In particular note that you can still pass `copycols` keyword argument
+to `DataFrame!` to override its default value.
```

Comment:
but please don't as that would be hella confusing?

Reply:
Yeah, probably better throw an error if `copycols` is provided. Throwing is always safer since we can allow it later if needed without breaking the API.

Reply:
OK